crypto/dh/dh_group_params.c

```diff
@@ -81,6 +81,17 @@ void dh_cache_named_group(DH *dh)
     }
 }
 
+int dh_is_safeprimegroup(const DH *dh)
```

Comment:
I'd suggest naming the check `dh_is_known_safe_prime_group`. As this is really only about known groups and not any arbitrary safe prime groups.

Reply:
what about is named instead of known?

Reply:
Sure, that works as well.